Sources/AppStoreConnectCLI/Commands/TestFlight/Sync/TestFlightProgramDifference.swift

```diff
@@ -52,9 +52,20 @@ struct TestFlightProgramDifference {
         }
     }
 
+    enum Error: LocalizedError, Equatable {
```

Comment:
There's an argument to be made around not adding equatable conformance in the implementation if you only use it for testing. Not sure how I feel about it though

Reply:
> There's an argument to be made around not adding equatable conformance in the implementation if you only use it for testing. Not sure how I feel about it though

Any ideas for removing the Equatable in this case? 🤔

Reply:
If you wanted to you could replace the instances in testing with something like:
```swift
        XCTAssertThrowsError(try TestFlightProgramDifference(local: local, remote: remote)) { error in
            guard case TestFlightProgramDifference.Error.duplicateTesters(let email) = error else {
                return XCTFail()
            }

            XCTAssertEqual(email, "foo@gmail.com")
        }
```